README.md

```diff
@@ -478,6 +478,13 @@ These are the available config options for making requests. Only the `url` is re
     
     // throw ETIMEDOUT error instead of generic ECONNABORTED on request timeouts
     clarifyTimeoutError: false,
+  },
+
+  // `defaultReject`, when set to a function, is executed when your axios request is rejected
+  // and you haven't explicitly used a catch block
+  // it implements the unhandledRejection event
+  defaultReject: function (err) {
+    console.log(err.toJSON());
```

Comment:
maybe `console.error`?

also not sure about axios guidelines, but i would use full names
https://javascript.info/ninja-code#use-abbreviations

Reply:
also i think in order to not to break backward compatibility it makes sense to `throw error` here by default

Reply:
Makes sense ! So far, by default, defaultReject is null